exporter/sapmexporter/README.md

```diff
@@ -32,6 +33,16 @@ during final translation.  Intended to be used in tandem with identical configur
   User should calculate this as `num_seconds * requests_per_second` where:
     - `num_seconds` is the number of seconds to buffer in case of a backend outage
     - `requests_per_second` is the average number of requests per seconds.
+- `correlation`
+  - `enabled` (default = true): Whether to enable span/metric correlation.
```

Comment:
Is this default=true if the user omits the `correlation` block entirely or only if it is present?  Whatever it is this doc should clarify.